packages/wrangler/src/pipelines/cli/sinks/create.ts

```diff
@@ -102,6 +103,8 @@ export const pipelinesSinksCreateCommand = createCommand({
 		},
 	},
 	validateArgs: (args) => {
+		validateEntityName("sink", args.sink);
```

Comment:
The `validateEntityName` throws an UserError, but it should throw a `CommandLineArgsError` when used in `validateArgs`.

Reply:
Changed.